cmd/edgectl/aes_install.go

```diff
@@ -365,6 +367,8 @@ func (i *Installer) GetInstalledImageVersion() (string, error) {
 
 // Perform is the main function for the installer
 func (i *Installer) Perform(kcontext string) error {
+	ctx := context.TODO()
```

Comment:
There's `i.ctx` that will be canceled if the user hits Ctrl-C.

Reply:
:+1: